src/doc_table.h

```diff
@@ -79,12 +81,22 @@ int DocTable_SetByteOffsets(DocTable *t, t_docId docId, RSByteOffsets *offsets);
 RSPayload *DocTable_GetPayload(DocTable *t, t_docId dodcId);
 
 /** Get the docId of a key if it exists in the table, or 0 if it doesnt */
-t_docId DocTable_GetId(DocTable *dt, const char *key);
+t_docId DocTable_GetId(DocTable *dt, RSDocumentKey key);
+static inline t_docId DocTable_GetIdR(DocTable *dt, RedisModuleString *key) {
```

Comment:
Since creating a document key from a redis string doesn't require allocation, are these functions really needed? Wouldn't it be simpler to just make a macro or inline that converts a redis string to document key?

Reply:
leftover. I'll change it.